gcp_airflow_foundations/operators/gcp/create_dataset.py

```diff
@@ -0,0 +1,72 @@
+from typing import Optional, Sequence
+
+from airflow.models import BaseOperator
+
+from google.cloud.bigquery.dataset import Dataset
+from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook
+
+
+class CustomBigQueryCreateEmptyDatasetOperator(BaseOperator):
```

Comment:
What's wrong with the out of box operator? BigQueryCreateEmptyDatasetOperator

Reply:
The project id issue?

Reply:
Exactly